sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -718,6 +718,11 @@ public partial class BlobServiceStatistics
         internal BlobServiceStatistics() { }
         public Azure.Storage.Blobs.Models.BlobGeoReplication GeoReplication { get { throw null; } }
     }
+    public enum BlobSetExpiryRelativeTo
+    {
+        BlobCreationTime = 0,
+        CurrentTime = 1,
```

Comment:
We should make this `Now` to match `DateTime`/`DateTimeOffset.Now`.

Reply:
Changed to ```Now``` in both Blobs and Data Lake.